values.yaml

```diff
@@ -1,9 +1,14 @@
 # Common configurations
 
 imageCredentials:
+  create: false
```

Comment:
Shouldn't the default be `true` to remain backwards compatible?

Reply:
Done

Reply:
Done